pyaedt/edb_core/components.py

```diff
@@ -1108,35 +1110,43 @@ def add_port_on_rlc_component(self, component=None):
             pt = self._pedb.point_data(*pos_pin_loc)
 
             pin_layers = self._padstack._get_pin_layer_range(pins[0])
-            pos_pin_term = self._pedb.edb_api.cell.terminal.PointTerminal.Create(
+            pos_pin_term = self._pedb.edb_api.cell.terminal.PadstackInstanceTerminal.Create(
                 self._active_layout,
                 pins[0].GetNet(),
                 "{}_{}".format(component.refdes, pins[0].GetName()),
-                pt,
+                pins[0],
                 pin_layers[0],
+                False,
             )
             if not pos_pin_term:  # pragma: no cover
                 return False
             neg_pin_loc = self.get_pin_position(pins[1])
             pt = self._pedb.point_data(*neg_pin_loc)
 
-            neg_pin_term = self._pedb.edb_api.cell.terminal.PointTerminal.Create(
+            neg_pin_term = self._pedb.edb_api.cell.terminal.PadstackInstanceTerminal.Create(
```

Comment:
You use pin[0].get_terminal() here.

Reply:
as agreed in private discussion closing it